File: The Vowel Code.ts
interface arr  {
  [key : string] : string;
}
// turn vowels into numbers
export function encode(string: string): string {
  const A:arr = {"a":"1","e":"2","i":"3","o":"4","u":"5"}
  return string.split("").map(v => A[v] || v).join("")
}

// turn numbers back into vowels
export function decode(string: string): string {
  const A:arr = {"1":"a","2":"e","3":"i","4":"o","5":"u"}
  return string.split("").map(v => A[v] || v).join("")
}

File: The Vowel Code.py
vowel = "aeiou"
def encode(st):
    return "".join([str(vowel.index(s)+1) if s in vowel else s for s in st ])
    
def decode(st):
    fil = list(map(lambda x:str(x),range(1,6)))
    return "".join([vowel[int(s)-1] if s in fil else s for s in st ])
